docs/application/native/guides/app-management/component-manager.md

```diff
@@ -0,0 +1,295 @@
+# Component Manager
+
+Component manager provides information about installed and running components.
+
+The main features of Component Manager API include:
+
+- Managing component context
+
+  You can retrieve the running component context and operate on it. The `component_context_h` handle of Component Context API (in [mobile](../../api/mobile/latest/group__CAPI__COMPONENT__CONTEXT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__COMPONENT__CONTEXT__MODULE.html) applications) is related to the running components and can be used to [manage the component context](#manage_context).
+
+- Managing component information
+
+  The `component_info_h` handle of Component Information API (in [mobile](../../api/mobile/latest/group__CAPI__COMPONENT__INFO__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__COMPONENT__INFO__MODULE.html) applications) is related to the available components that are installed but not necessarily running.
+
+Iterator functions are used to travel through a list of components. `component_manager_foreach_component_context()` is used in running components and `component_manager_foreach_component_info()` is used in available components. Each function calls a callback function `component_manager_component_context_cb()` or `component_manager_component_info_cb()`, passing the handle for each component.
+
+## Prerequisites
+
```

Comment:
Suggestion: Can we add a leading sentence as it is there in PR #841:

To enable your application to use the component management functionality:

Reply:
Done